tensorflow/core/kernels/stateful_random_ops_gpu.cu.cc

```diff
@@ -130,4 +138,4 @@ template struct UpdateVariableAndFill_Philox<
 
 }  // end namespace tensorflow
 
-#endif  // GOOGLE_CUDA
+#endif  // GOOGLE_CUDA || TENSORFLOR_USE_ROCM
```

Comment:
nit picking: typo

Reply:
will fix this, the next time I need to commit something (which hopefully wont be for this PR!)